templates/rhdm71-prod-kieserver.yaml

```diff
@@ -0,0 +1,916 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  annotations:
+    iconClass: icon-jboss
+    tags: rhdm,jboss,prod
+    version: "1.0"
+    openshift.io/display-name: Red Hat Decision Manager 7.1 production environment
+    openshift.io/provider-display-name: Red Hat, Inc.
+    description: Application template for a managed HA production runtime environment, for Red Hat Decision Manager 7.1
+    template.openshift.io/long-description: This template defines resources needed for a HA prouction runtime environment for Red Hat Decision Manager 7.1, including application deployment configuration, secure and insecure http communication and persistent volume to store data that needs to survive to a restart.
+    template.openshift.io/documentation-url: https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.1/html/deploying_a_red_hat_decision_manager_7.1_authoring_environment_on_red_hat_openshift_container_platform/
+    template.openshift.io/support-url: https://access.redhat.com
+    template.openshift.io/bindable: "false"
+  name: rhdm71-prod
+labels:
+  template: rhdm71-prod
+  rhdm: "1.0"
+message: |-
+          A new environment has been set up for Red Hat Decision Manager 7.
+          Both sets of KIE servers are configured with the username/password
+
+              Username: ${KIE_SERVER_USER}
+              Password: ${KIE_SERVER_PWD}
+
+          Please be sure to create the secret named "${KIE_SERVER_HTTPS_SECRET}" containing the
+          ${KIE_SERVER_HTTPS_KEYSTORE} files used for serving secure content.
+parameters:
+- displayName: Application Name
+  description: The name for the application.
+  name: APPLICATION_NAME
+  value: myapp
+  required: true
+- displayName: Maven repository URL
+  description: Fully qualified URL to a Maven repository or service.
+  name: MAVEN_REPO_URL
+  example: http://nexus.nexus-project.svc.cluster.local:8081/nexus/content/groups/public/
+  required: true
+- displayName: Maven repository username
+  description: Username to access the Maven repository, if required.
+  name: MAVEN_REPO_USERNAME
+  required: false
+- displayName: Maven repository password
+  description: Password to access the Maven repository, if required.
+  name: MAVEN_REPO_PASSWORD
+  required: false
+- displayName: Name of the Maven service hosted by Decision Central
+  description: The service name for the optional decision central, where it can be reached, to allow service lookups (for maven repo usage), if required
+  name: DECISION_CENTRAL_MAVEN_SERVICE
```

Comment:
Maybe rename to KIE_SERVER_MAVEN_SERVICE as this template does not include the decision central?

Reply:
Will use INTERNAL_MAVEN_SERVICE and INTERNAL_MAVEN_REPO_PATH